recipes/diamond_add_taxonomy/run_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export LC_ALL=en_US.utf-8
```

Comment:
This is tricking the container testing, it will create an unfunctional container. If you put this back into the meta.yaml file it will probably fail.

Reply:
Ah ok, fixed using extended-base container.